tests/slow_tests/memory_test/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+file(GLOB SOURCE_FILES "*.cpp")
+add_wast_target(memory_test "${SOURCE_FILES}" "${CMAKE_SOURCE_DIR}/contracts" ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
This directive will build the memory test contract code intp `contracts/` however you later attempt to include the `.wast.hpp` file relative to the slow tests source.

we have precedence for storing "test" contracts in the contracts tree that are used in slow tests.  the `memory_test` contract should be placed there (along side `api_test`).  Then you can include the `.wast.hpp` using brackets and it should work just fine.